docs/source/gbd2017_models/risk_attributable_causes/iron_deficiency/index.rst

```diff
@@ -151,9 +153,172 @@ described in the table above.
 Iron Deficiency Risk Factor
 +++++++++++++++++++++++++++
 
-The iron deficiency risk factor in GBD 2017 is a **continuous measure** modeled 
-as a distribution of hemoglobin concentrations *among the population with 
-conditions that cause iron deficiency anemia.* Below is a list of such causes, along with the list of sequela IDs that include anemia health states:
+In GBD 2017, the iron deficiency risk factor is used for two applications. The 
+first is the PAF of 1 relationship with the dietary iron deficiency anemia 
+cause and the second is a risk outcome relationship with maternal disorder 
+causes. 
+
+Notably, the iron deficiency risk factor in GBD 2017 represents the **age-, 
+sex-, and location-specific mean hemoglobin concentration among the total 
+population.** The mean value for the iron deficiency risk factor is 
+stored under modelable entity ID 10487 (also REI ID 95) and the standard 
+deviation is stored under modelable entity ID 10488. The iron deficiency risk 
+factor (population hemoglobin concentration) follows a 40% gamma and 60% mirror 
+Gumbel ensemble distribution. 
+
+NOTE:
+
+	The values stored in the iron deficiency risk factor (i.e. population  
+	hemoglobin concentration parameters) are used in the GBD modeling process 
+	to calculate risk-deleted population hemoglobin concentration where the 
+	risk is all iron deficiency (i.e. iron responsive anemias), which serves as 
+	the population TMREL, in order to calculate the population attributable 
+	fraction between the iron deficiency risk factor and maternal disorder 
```

Comment:
This sentence is a bit hard to follow because it's so long.  Perhaps split it into a couple sentences?